hydra-node/src/Hydra/Chain/Direct/Handlers.hs

```diff
@@ -67,40 +73,42 @@ import System.IO.Error (userError)
 data LocalChainState m = LocalChainState
   { getLatest :: STM m ChainStateAt
   , pushNew :: ChainStateAt -> STM m ()
```

Comment:
To make this consistent, we should also use `ChainStateType Tx` in the `getLatest` and `pushNew` functions.